api.md

```diff
@@ -0,0 +1,392 @@
+<div align="center">
```

Comment:
No mention of the word "test" in this file: https://github.com/dwyl/imgup/blame/api-%2353/api.md
<img width="1071" alt="no-tests-in-api-file" src="https://github.com/dwyl/imgup/assets/194400/20eb569e-e2a8-487e-97d8-e324b451c33b">

By definition this is incomplete.

Reply:
Testing instructions added: https://github.com/dwyl/imgup/blob/40e95f493c926f21cb5877ed456fba01cc9e13ca/api.md#22-test-the-upload1-function